src/linechart/DataSeries.jsx

```diff
@@ -12,129 +11,90 @@ module.exports = React.createClass({
 
   propTypes: {
     data: React.PropTypes.array,
-    seriesName: React.PropTypes.string,
-    interpolationType: React.PropTypes.string,
     fill: React.PropTypes.string,
```

Comment:
The `fill` props is no longer needed since we are using `colors` and `colorAccessor`